docs/deploy.rst

```diff
@@ -63,6 +63,9 @@ Nginx can be used by adding a block such as:
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto $scheme;
         proxy_pass http://localhost:9000/;
+        # Must turn off response buffering for SSE to work
+        proxy_buffering off;
+        proxy_read_timeout 600s;
```

Comment:
Ah, yes, I will add that as we probably do need it since no data will be sent over the channel.